### PR TITLE
Italian translation

### PR DESCRIPTION
**Pull request by AuryPolaris**
Here's my translation of the current Puroto's blog

### CODE REVIEW

**Review by staxfur — APPROVED** [review]
Thank you really much!

File: blogs/introduction/it/introduction.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+Puroto è una community contenente una grande varietà di persone provenienti da paesi di tutto il mondo. Anche lo staff consiste in persone provenienti da Germania, Francia, Stati uniti e molti altri paesi; avere un team così esteso attraverso il mondo ci permette di offrire questo blog in più lingue. A Puroto noi pensiamo che tutti debbano sentirsi come a casa a Puroto e stiamo facendo del nostro meglio affinchè questo si realizzi.
+
+Stiamo facendo del nostro meglio nel caricare nuovi post con le loro traduzioni mentre li pubblichaimo, ma visto che non abbiamo il backup di alcune lingue, dovremo aggiungerle dopo la pubblicazione qualche volta. Grazie per la comprensione!
+
+>La tua lingua non è ancora disponibile? sentiti libero di [candidarti come traduttore per il nostro blog](https://apply.puroto.net/)! Traduzioni una tantum di post esistenti sono apprezzate lo stesso!
+
+# In conclusione
+Speriamo che siate eccitati per i prossimi contenuti che arriveranno qui e per il futuro di Puroto. Daremo il massimo per lavorare il più vicino possibile con voi, la nostra community ed informarvi sul progetto! Unitevi a noi per far diventare la comunità furry fantastica!
+
+
```
Comment (Alekuso) [inline blogs/introduction/it/introduction.md]: I suggest you to only put one blank line between contents in Markdown. It's better for the file formatting.

Comment (AuryPolaris) [reply blogs/introduction/it/introduction.md]: oki I am modifing it now!

Comment (AuryPolaris) [reply blogs/introduction/it/introduction.md]: just done

**Review by Alekuso — APPROVED** [review]
Looks good to me!

**Pull request merged**